frontend/src/pages/acceleratorProfiles/screens/list/AcceleratorProfilesTable.tsx

```diff
@@ -5,35 +5,58 @@ import { Table } from '~/components/table';
 import AcceleratorProfilesTableRow from '~/pages/acceleratorProfiles/screens/list/AcceleratorProfilesTableRow';
 import { AcceleratorKind } from '~/k8sTypes';
 import { columns } from '~/pages/acceleratorProfiles/screens/list/const';
+import DeleteAcceleratorProfileModal from './DeleteAcceleratorProfileModal';
 
 type AcceleratorProfilesTableProps = {
-  accelerators: AcceleratorKind[];
+  acceleratorProfiles: AcceleratorKind[];
+  refreshAcceleratorProfiles: () => void;
 };
 
-const AcceleratorProfilesTable: React.FC<AcceleratorProfilesTableProps> = ({ accelerators }) => {
+const AcceleratorProfilesTable: React.FC<AcceleratorProfilesTableProps> = ({
+  acceleratorProfiles,
+  refreshAcceleratorProfiles,
+}) => {
   const navigate = useNavigate();
+  const [deleteAcceleratorProfile, setDeleteAcceleratorProfile] = React.useState<
+    AcceleratorKind | undefined
+  >();
```

Comment:
Only need to pass accelerator profile name

Reply:
no we need the whole accelerator, because the modal relies in the display name and the metadata name